tests/unit_test/action/action_test.py

```
@@ -1386,10 +1484,16 @@ def _get_action(*args, **kwargs):
         responses.add(
-            "POST", Utility.environment['evaluator']['pyscript']['url'],
-            json={"success": True, "data": {"bot_response": "Successfully Evaluated the pyscript",
-                                            "slots": "invalid slots values", "type": "text"},
-                  "message": None, "error_code": 0},
+            "POST", Utility.environment['async_callback_action']['pyscript']['url'],
+            json={
+                "success": True,
+                "body": {
+                    "bot_response": "Successfully Evaluated the pyscript",
+                    "slots": "invalid slots values", "type": "text",
+                    "type": "text"
```

Ruff (0.8.2)

Dictionary key literal `"type"` repeated

Remove repeated key literal `"type"`

(F601)